src/directives/disable-container.js

```diff
@@ -0,0 +1,82 @@
+/*
+Copyright 2025 ODK Central Developers
+See the NOTICE file at the top-level directory of this distribution and at
+https://github.com/getodk/central-frontend/blob/master/NOTICE.
+
+This file is part of ODK Central. It is subject to the license terms in
+the LICENSE file found in the top-level directory of this distribution and at
+https://www.apache.org/licenses/LICENSE-2.0. No part of ODK Central,
+including this file, may be copied, modified, propagated, or distributed
+except according to the terms contained in the LICENSE file.
+*/
+
+import vTooltip from './tooltip';
+
+
+const createOverlay = () => {
+  const overlay = document.createElement('div');
+  overlay.classList.add('disable-container-overlay');
+  overlay.setAttribute('tabindex', '0'); // make it focusable with keyboard
+  vTooltip.mounted(overlay, {
+    modifiers: { 'aria-describedby': true },
+    value: 'Bulk operation in progress' // TODO: translation
+  });
+
+  return overlay;
+};
+
+const setInertOnChildren = (container, inert) => {
```

Comment:
It would be nice to avoid this and set insert on the container but if I do that then it would be impossible to make overlay focusable and show tooltip on it